src/script/tracking/EventTrackingRepository.ts

```diff
@@ -226,24 +231,27 @@ export class EventTrackingRepository {
   private async startProductReporting(trackingId?: string): Promise<void> {
     // This is a global object provided by the countly.min.js script
     if (!isCountlyLoaded()) {
+      this.countlyLogger.warn('Countly is not available');
       return;
     }
 
     if (!isCountlyEnabledAtCurrentEnvironment() || this.isProductReportingActivated) {
+      this.countlyLogger.warn('Countly is not enabled at this environment');
       return;
     }
     this.isProductReportingActivated = true;
 
-    const {isProduction} = getWebEnvironment();
-
     // Add Parameters to previous Countly object
 
-    window.Countly.app_version = Config.getConfig().VERSION;
-    window.Countly.debug = !isProduction;
+    const {COUNTLY_ENABLE_LOGGING, VERSION} = Config.getConfig();
+
+    window.Countly.app_version = VERSION;
+    window.Countly.debug = COUNTLY_ENABLE_LOGGING;
```

Comment:
I mean this as a guarantee countly logging doesn't happen on prod environments, even if a mistake happens with the env var

```suggestion
    window.Countly.debug = COUNTLY_ENABLE_LOGGING && !isProduction;
```

Reply:
Actually i believe this setting has no effect in any way.

Since countly doesnt react on this variable change dynamically and it is set the first time in the html files